frontend/src/components/Application/Posts/views/PostEditor.vue

```diff
@@ -2,23 +2,71 @@
   <div class="container mt-2">
     <h1 class="mb-3">Post Something</h1>
     <v-form @submit.prevent="onSubmit">
-      <v-textarea
-        v-model="post_body"
-        class="form-control"
-        placehodler="What do you want to post?"
-        rows="3"
-      ></v-textarea>
+      <v-card>
+        <v-card-text>
+          <v-layout row wrap justify-space-around>
+            <v-flex xs8 sm9 text-xs-center>
+              <v-text-field
+                v-model="post_body"
+                class="form-control"
+                placeholder="What do you want to post?"
+                rows="3"
+              ></v-text-field>
+              <p v-if="error" class="grey--text">{{ error }}</p>
+              <p v-else class="mb-0">
+                <v-text-field v-model="speech" v-if="sentences.length > 0" v-bind:key="sentences" 
```

Comment:
You're seeing the recorded speech placed in this text field because it's being modeled from the 'speech' variable (v-model="speech") instead of the 'post_body' variable. It may be possible to model it on the 'post_body' variable instead, but it would have to be coupled with a bunch of other changes. I'd just leave it alone for now.

Reply:
Yes, I had to do that because it caused the post_body textfield to disappear until speech-to-text was used.